### PR TITLE
Fix tls in CI: Use openssl 1.0.2 in CentOS and buster

### PR DESCRIPTION
**Pull request by bwoebi**
### Description

Openssl 1.0.1 in unable to properly handle letsencypt certificates where a cross-signing root has expired, but packagist is using letsencrypt, thus composer became unusable.

### Readiness checklist
- ~~[ ] (only for Members) Changelog has been added to the release document.~~
- ~~[ ] Tests added for this feature/bug.~~

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

**Review by labbati — DISMISSED** [review]
Thank you @bwoebi! LGTM

File: dockerfiles/ci/centos/6/Dockerfile [hunk 1]
```diff
@@ -35,6 +43,10 @@ ENV SRC_DIR=/usr/local/src
 
 COPY download-src.sh /root/
 RUN set -eux; \
+# version 1.0.2 of openssl (default version is 1.0.1) required for letsencrypt certificates to continue working
```
[the remark this answers is not in the document]
Comment (bwoebi) [reply dockerfiles/ci/centos/6/Dockerfile:46]: No, due to the limitations of PHP5, which does not support openssl 1.1.x

[the remark this answers is not in the document]
Comment (labbati) [reply dockerfiles/ci/centos/6/Dockerfile:46]: 👍🏻  thanks

**Review by labbati — APPROVED** [review]
👍🏻  changes LGTM

**Pull request merged**